### PR TITLE
Adding a new setting to disable Setup tutorial

### PR DESCRIPTION
**Pull request by cartergreen**
ALSO is it possible to alphabetize this list, similar to the one that we have for the Windows policy list.  The order of this list seems Wild, I dont know markdown super well so I'm assuming someone can do it faster than me.  I'd like to alphabetize the list based on the Settings column.

### CODE REVIEW

**Comment by cartergreen** [comment]
@kaarins @gacarini

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/deploy-and-configure-on-macos.md [hunk 1]
```diff
@@ -74,6 +74,7 @@ Use the following keys to preconfigure or change settings for your users. The ke
...
 |Block External Sync|Prevents the sync app from syncing libraries and folders shared from other organizations.|BlockExternalSync (Bool): Set to true to prevent syncing OneDrive and SharePoint libraries and folders from organizations other than the user's own organization. Set to false or do not include the setting to allow.<br/>[Learn about OneDrive B2B Sync](b2b-sync.md).|\<key\>BlockExternalSync\</key\><br/>\<(Bool)/\>|
+|Disable the tutorial that appears at the end of OneDrive Setup|This setting lets you prevent the tutorial from showing at the end of OneDrive Setup.|DisableTutorial (Bool): When set to true, Users will not see the tutorial after they complete OneDrive Setup.|\<key\>DisableTutorial\</key\><br/>\<(Bool)/\>
```
Comment (JohanFreelancer9) [inline OneDrive/deploy-and-configure-on-macos.md:77]:
```suggestion
|Disable the tutorial that appears at the end of OneDrive Setup|This setting lets you prevent the tutorial from showing at the end of OneDrive Setup.|DisableTutorial (Bool): When set to true, users will not see the tutorial after they complete OneDrive Setup.|\<key\>DisableTutorial\</key\><br/>\<(Bool)/\>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**